scripts/HFIR_4Circle_Reduction/message_dialog.py

```diff
@@ -7,12 +7,13 @@
 # Dialog for message
 from __future__ import (absolute_import, division, print_function)
 from six.moves import range
-from PyQt4 import QtGui, QtCore
+from qtpy.QtWidgets import (QDialog)  # noqa
+
 
 from . import ui_messagebox
 
 
-class MessageDialog(QtGui.QDialog):
```

Comment:
Does this need to be a separate class? It looks like a free function kicking off `QMessageBox` would cover it.